discojs/discojs-core/src/tasks/simple_face.ts

```diff
@@ -1,4 +1,4 @@
-import { dataset, Task } from '..'
+import { data, Task } from '..'
```

Comment:
does this task actually need `data`? the other tasks don't

Reply:
we have 3 tasks using data preprocessing (lus-covid, geotags, simple-face), but since we're within the disco lib, there are multiple ways to import the `ImagePreprocessing` enum used by all three classes

for instance, lus-covid and geotags import the enum directly from `src/dataset/index.ts`, wheras simple-face imports the `data` namespace from `src/index.ts` and then access the enum

this makes no difference, but I'll modify geotags and lus-covid to import the enum in the standard way